lib/elixir/lib/enum.ex

```diff
@@ -1408,7 +1408,14 @@ defmodule Enum do
 
   """
   @spec sum(t) :: number
-  def sum(enumerable) do
```

Comment:
He's talking about a body less clause. By your current version, the docs do name the argument automatically, if you have the bodyless clause, it will appear as `enumerable` in the docs.